src/main/java/nl/tudelft/cse1110/andy/execution/metatest/evaluators/StringReplacementEvaluator.java

```diff
@@ -0,0 +1,25 @@
+package nl.tudelft.cse1110.andy.execution.metatest.evaluators;
+
+public class StringReplacementEvaluator implements MetaEvaluator {
+
+    private String old;
+    private String replacement;
+
+    public StringReplacementEvaluator(String old, String replacement) {
+        this.old = old;
+        this.replacement = replacement;
+    }
+
+    @Override
+    public String evaluate(String oldLibraryCode) {
+        String shiftedOld = this.old.replaceAll("(?m)^\\s+", "");
+        String shiftedOldLibraryCode = oldLibraryCode.replaceAll("(?m)^\\s+", "");
+        String replaced = shiftedOldLibraryCode.replace(shiftedOld, this.replacement);
+
+        if (replaced.equals(oldLibraryCode)) {
+            throw new RuntimeException("Meta test failed to find this text replacement:\n" + this.old);
```

Comment:
Just something I remembered: When I was converting assignments in September, there was a bug with this in previous versions of Andy and this runtime exception is actually suppressed later, so it simply causes the meta test to fail instead of raising a generic failure, which is a bit confusing. I'll see if there's a clean way of fixing that (if it hasn't been fixed already due to the refactoring).

Reply:
Please, open an issue about it. If there's an exception in the meta tests, we should fail the whole execution.

Reply:
Fixed in #91